pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -254,7 +254,9 @@ stdenv.mkDerivation (finalAttrs: {
 
   # Add a ‘qemu-kvm’ wrapper for compatibility/convenience.
   postInstall = lib.optionalString (!toolsOnly) ''
```

Comment:
Could we make this depend on an option we have? I think that would be nicer, to not accidentally drop it in some update.